spring-integration-core/src/test/java/org/springframework/integration/config/AggregatorParserTests.java

```diff
@@ -266,5 +302,19 @@ public void testAggregatorFailureIfMutuallyExclusivityPresent() {
 		return MessageBuilder.withPayload(payload).setCorrelationId(correlationId).setSequenceSize(sequenceSize)
 				.setSequenceNumber(sequenceNumber).setReplyChannel(outputChannel).build();
 	}
-	
+
+	public static class MyMGP extends SimpleMessageGroupProcessor {
```

Comment:
Well, we really have a mix of names (`SimpleMessageStore`, `SimplePatternFileListFilter`, `PassThruLockRegistry`, `PassThruMessageConverter` etc..).

As usual: there is need to do a task to refactor the code via proper way :smile: .

OK. Let's deprecate that one and leave this new one as is!
(Although its JavaDocs require some polishing in that case :smile: )

I've introduced `PassThruMessageConverter`, because we already have several `SimpleMessageConverter` :disappointed: